gTea/app/src/main/java/com/nechaieva/gtea/utils/LevenshteinDistCalculator.java

```diff
@@ -1,6 +1,10 @@
 package com.nechaieva.gtea.utils;
 
 public class LevenshteinDistCalculator {
+
+    private LevenshteinDistCalculator() {
```

Comment:
This is a great example (again!) that our current review process with the single simple cl's does not work. I flagged it while going through your cl even though you addressed it already in a future cl....